### PR TITLE
Harden native interop and repository foundations

### PR DESCRIPTION
**Pull request by JeremyKuhne**
## Summary

- harden clipboard memory allocation, native error handling, and ownership transfer
- fix COM, WIC, `UNICODE_STRING`, and registry edge cases with focused tests using Touki TestSupport for private access
- add security, contribution, conduct, issue, and pull-request guidance for the pre-release repository

## Validation

- Debug and Release solution builds
- Debug and Release tests: 286 passed, 1 manual test skipped in each configuration
- Markdown lint and link validation
- NuGet vulnerability audit: no known vulnerable packages

### CODE REVIEW

File: src/thirtytwo/Clipboard.cs [hunk 1]
```diff
@@ -118,16 +118,52 @@ public static uint RegisterClipboardFormat(string formatName)
...
+                data.CopyTo(buffer);
+                buffer[^1] = default;
+            }
+            finally
+            {
+                // FALSE with NO_ERROR is expected when the lock count reaches zero.
+                PInvoke.GlobalUnlock(global);
+            }
+
+            HANDLE result = PInvoke.SetClipboardData(format, (HANDLE)(nint)global);
```
[the remark this answers is not in the document]
Comment (JeremyKuhne) [reply src/thirtytwo/Clipboard.cs:157]: Fixed in 2408f7b. The normal path now checks `GlobalUnlock`'s `FALSE` result and throws only for a non-`ERROR_SUCCESS` last error; the exception path performs a best-effort unlock and rethrows the original exception.
